samples/ipc_call_interface/consumers/my_consumer.c

```diff
@@ -7,13 +7,13 @@
 #include "az_ulib_query_1_model.h"
 #include "az_ulib_result.h"
 #include "azure/az_core.h"
-#include "wrappers/cipher_1_wrapper.h"
+#include "cipher_1_model.h"
 #include <inttypes.h>
 #include <stdint.h>
 #include <stdio.h>
 
-static az_ulib_ipc_interface_handle _cipher_1;
 #define BUFFER_SIZE 200 // Should be big enough to fit JSON input and output.
+#define KEY_VAULT_PACKAGE_NAME "key_vault"
```

Comment:
nit: shouldn't this just be `PACKAGE_NAME`?

Reply:
I think it is better to keep the full name, in case your application use more than one package. #Pending